app/controllers/RepresentativeDanImportVATController.scala

```diff
@@ -59,38 +60,46 @@ class RepresentativeDanImportVATController @Inject()(identify: IdentifierAction,
         backLink
       ))),
       dan => {
-        val previousVATAccountNumber = request.userAnswers.get(AdditionalDefermentNumberPage).getOrElse(dan.accountNumber)
-        val previousVATAccountType = request.userAnswers.get(AdditionalDefermentTypePage).getOrElse(dan.danType)
-        if (dan.accountNumber != previousVATAccountNumber || dan.danType != previousVATAccountType) {
+        val dutyAccountNumber = request.userAnswers.get(DefermentAccountPage)
```

Comment:
dutyAccountNumber only used in the if statement, could just have it val dutyAccountNumberExists = request.userAnswers.get(DefermentAccountPage).contains(dan.accountNumber)